extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/GoogleTransferExtension.java

```diff
@@ -100,7 +100,7 @@ public void initialize(ExtensionContext context) {
     exporterBuilder.put("calendar", new GoogleCalendarExporter(credentialFactory));
     exporterBuilder.put("mail", new GoogleMailExporter(credentialFactory));
     exporterBuilder.put("tasks", new GoogleTasksExporter(credentialFactory));
-    exporterBuilder.put("photos", new GooglePhotosExporter(credentialFactory));
+    exporterBuilder.put("photos", new GooglePhotosExporter());
```

Comment:
Curious, do we no longer need the credentialFactory? I thought this was for extending lifetime of oauth tokens if the transfer takes too long?

Reply:
Thanks for reminding me!